virtuoso-docs-source/xmlsource/rdfperformancetuning.xml

```diff
@@ -897,53 +877,50 @@ CREATE PROCEDURE dump_one_graph
       {
         http (' .\n', ses);
         string_to_file (file_name, ses, -1);
+        gz_compress_file (file_name, file_name||'.gz');
+        file_delete (file_name);
       }
   }
 ;
-</programlisting>
-            </section>
-            <section xml:id="rdfperfdumpandreloadgraphdumponegraphex">
-              <title>Example</title>
-              <orderedlist>
-                <listitem>
-                  <para>Call the </para>
-                  <para>
-                    <emphasis>dump_one_graph</emphasis>
-                  </para>
-                  <para> procedure with appropriate arguments:
-</para>
-                  <programlisting>
+          </programlisting>
+            <para>To load the procedure into Virtuoso, it can simply be copied and pasted into the Virtuoso <code>isql</code> command line tool or Interactive SQL UI of the Conductor and executed.</para>
```

Comment:
```suggestion
            <para>To load the procedure into Virtuoso, it can simply be copied, pasted, and executed, through the Virtuoso <code>isql</code> command line tool or the Interactive SQL UI of the Conductor.</para>
```